schema_salad/python_codegen.py

```diff
@@ -170,6 +170,8 @@ def begin_class(
             self.out.write("    pass\n\n\n")
             return
 
+        field_names.append("_doc")
```

Comment:
These are the fields of the model (which are closely related to, but not exactly the same, as the fields of the class being generated).  Why are you adding this here?

Reply:
I want the original CommentedMap that had line column information to be associated with the top level class (e.g., Workflow) in order to update the attributes that are being saved. I wasn't sure how to save this Commented Map besides saving it as a class attribute. The save method called by the user goes directly to the save method within the class, so I didn't see a way of passing the Commented Map to the first call to save. If you have any ideas of doing this an alternate way I would be happy to reformat it!

Reply:
I actually changed it to have a global doc variable and have the save methods take a list of strings representing what level in the doc that object is. This is useful for the second issue as well, since there are places that error messages need line numbers appended where no version of the doc is present.